backend/app/infrastructure/__init__.py

```diff
@@ -56,20 +60,62 @@ def __getattribute__(self, name):
         return attr
 
 
+def lookup_max_time():
+    assert max_time is not None
+    return max_time
+
+
+def is_graph_error_permanent(error):
+    # TODO this must differentiate between different sources reasons for
+    #      failure
+    # Figure out where exactly is string about reorgs is hidden
```

Comment:
can't it be done in this task? if not, please mark this as to-do as well

Reply:
I've tried to do it - triggering CI multiple times. Haven't had luck and haven't seen one TransportQueryError. We really need a task on CI that will run a test in a loop multiple times, until it fails.

Reply:
Fortunately, distinguishing between different types of TransportQueryError here is a microoptimisation. It only allows us to give up a bit earlier.